### PR TITLE
Add PNDC to Tokens and Price

### PR DESCRIPTION
**Pull request by timmaycastags**
This request is to add the contract for PNDC to the ETH Tokens and Price tables.

### CODE REVIEW

**Comment by timmaycastags** [comment]
I have read the CLA Document and I hereby sign the CLA

File: models/prices/ethereum/prices_ethereum_tokens_legacy.sql
Comment (jeff-dude) [inline models/prices/ethereum/prices_ethereum_tokens_legacy.sql:1]: for prices, we no longer use the legacy files / engine. please add this value to the file in the same directory that is non-legacy, dunesql version

you can then remove this change from this file 👍

**Review by jeff-dude — CHANGES_REQUESTED** [review]
no need to apologize for exploring spellbook -- thank you for contributing!
i've left one comment for change below

File: models/prices/ethereum/prices_ethereum_tokens_legacy.sql
Comment (timmaycastags) [reply models/prices/ethereum/prices_ethereum_tokens_legacy.sql:1]: Thanks Jeff,  The non-legacy price file already has the pndc coin added.  I will remove the legacy update and no change for price is needed.

Comment (jeff-dude) [reply models/prices/ethereum/prices_ethereum_tokens_legacy.sql:1]: sounds good!
looks like your second commit re-added it back. please do look to remove, so there are only two files in the PR (tokens files)

**Pull request closed without merging**